google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -1601,7 +1603,7 @@ def reload!
           @gapi = reloaded_gapi
           self
         end
-        alias_method :refresh!, :reload!
+        alias refresh! reload!
```

Comment:
Is `alias` preferred over `alias_method` by the new rubocop? The two do have subtly different semantics (notably, if you override `reload!`, now `refresh!` will still point at the old method).

Reply:
Here's the reasoning: https://github.com/bbatsov/ruby-style-guide#alias-method-lexically

Reply:
Yes, this was changed in the the updated style guide in the more recent Rubocop.